docs/user/exams/instructors_guide.rst

```diff
@@ -506,6 +510,20 @@ You can specify the moment when Artemis publishes the results of the exam, see `
 
    Exam Scores Page
 
+4.2. Exam Solutions
+^^^^^^^^^^^^^^^^^^^
+Optionally, you can publish the example solutions of text, modeling, file upload and programming exercises to the participated students after a desired date by setting ``Example Solution Publication Date`` of the exam to a non-empty date.
```

Comment:
```suggestion
Optionally, you can publish the example solutions of text, modeling, file upload and programming exercises to the registered students after a desired date by setting ``Example Solution Publication Date`` of the exam to a non-empty date.
```
participated seams off here. If "registered" is not correct, you should find some other wording.

Reply:
I think this sounds the best.
```suggestion
Optionally, you can publish the example solutions of text, modeling, file upload and programming exercises to students with submissions after a desired date by setting ``Example Solution Publication Date`` of the exam to a non-empty date.
```